src/sql-parser/src/parser.rs

```diff
@@ -3636,12 +3636,11 @@ impl<'a> Parser<'a> {
         if self.parse_keyword(INCLUDE) {
             self.parse_comma_separated(|parser| {
                 let ty = match parser
-                    .expect_one_of_keywords(&[KEY, TIMESTAMP, PARTITION, TOPIC, OFFSET, HEADERS])?
+                    .expect_one_of_keywords(&[KEY, TIMESTAMP, PARTITION, OFFSET, HEADERS])?
                 {
                     KEY => SourceIncludeMetadataType::Key,
                     TIMESTAMP => SourceIncludeMetadataType::Timestamp,
                     PARTITION => SourceIncludeMetadataType::Partition,
-                    TOPIC => SourceIncludeMetadataType::Topic,
```

Comment:
this isnt backwards compatible, are we just going to check against prod during the release?

Reply:
It is because this was always denied in the planner so there aren't any sources that have this. The only that this changes is the error message you get. Note that `INCLUDE TOPIC` is not even documented

Reply:
nice!